libraries/standard/mqtt/cbmc/proofs/Makefile-template-defines

```diff
@@ -0,0 +1 @@
+SRCDIR ?= $(abspath $(PROOF_ROOT)/../../../../..)
```

Comment:
I think this file is not required and is generated by `setup.py`

Reply:
i didnt delete it because it wasn't linked